components/DefaultButton.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { StyleSheet, Text, View, TouchableOpacity } from "react-native";
+import Color from "../constants/Colors";
+
+const DefaultButton = props => {
+  return (
+    <TouchableOpacity
+      style={{ ...styles.button, ...props.style }}
+      onPress={props.onPress}
+      activeOpacity={0.5}
```

Comment:
I think 0.7/0.8 looks nicer. 0.5 is abit too much. but your decision

Reply:
Actually the default is 0.2 so I increased it to 0.5.
If it'll look ugly, we'll change it later..